tests/api.test.ts

```diff
@@ -0,0 +1,213 @@
+import { codes } from '@adgorithmics/graphql-errors';
```

Comment:
I thought we were putting the test files in the same directory as the tested file.

Reply:
it makes it easier for npm to just include `"/src/**/*.d.ts",`

if we have the tests in `src` it'll also publish the definition files for those. And it doesn't support regex, its only glob.
And files field overwrites `.npmignore`

its a mess, so i just put them outside